frontend/assets/javascripts/src/modules/campaignCode.es6

```diff
@@ -0,0 +1,36 @@
+/**
+ * Used for storing campaign codes in a cookie for readout server-side.
+ * Necessary for recording campaigns in the soulmates dashboard.
+ *
+ */
+
+import { setCookie } from 'src/utils/cookie';
+import URLSearchParams from 'URLSearchParams';
+
+// Checks the querystring for INTCMP (internal campaign) and records in cookie.
+function recordCampaign () {
+
+	let campaignCode;
+
+	// Don't want JS to break if there is a problem reading query params.
+	// Just don't set the cookie if this happens.
+	try {
+
+		let urlParams = new URLSearchParams(window.location.search);
```

Comment:
Ok so it looks like this was necessary on older versions of the implementation, but not the most recent one (hence why it works for us). Unfortunately this means that there may be a few users with this older implementation who won't get picked up by the polyfill. So I've added the slice: 00026fe.